scripts/actions/abilities/pets/regal_scratch.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------
+-- Regal Scratch
+-- M=3,1
```

Comment:
what's this `M=3,1`? doesn't really bother me that it's in there i guess

Reply:
ha good question, was a direct copy from what we had on wings... i honestly have no idea